### PR TITLE
Feature/pipelines/new ref pipelines

### PR DESCRIPTION
**Pull request by swittenberger**

### CODE REVIEW

File: .pipelines/azure-pipelines/azure-pipelines-REFramework_Testing.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  #   - develop
+  #   - master/REFramework_Testing/*
+  # paths:
+  #   include:
+  #   - REFramework_Testing/*
+  #   - .pipelines/*
+
+
+variables:
+  - template: .pipelines/variables/common.yml
```
Comment (catalin-marin) [inline .pipelines/azure-pipelines/azure-pipelines-REFramework_Testing.yml]: nok, location is relative to your current location 
../variables/common.yml

File: .pipelines/azure-pipelines/azure-pipelines-REFramework_Testing.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  - name: projectPath
+    value: 'REFramework_Testing'
+  - name: projectNuspec
+    value: 'UiPath.Template.REFramework_Testing.nuspec'
+
+jobs:
+  - template: .pipelines/jobs/ci.yml
```
Comment (catalin-marin) [inline .pipelines/azure-pipelines/azure-pipelines-REFramework_Testing.yml]: nok, location is relative to your current location

File: DocumentUnderstandingProcess/UiPath.Template.DocumentUnderstandingProcess.nuspec [hunk 3]
```diff
@@ -2,7 +2,7 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
   <metadata>
     <id>UiPath.Template.DocumentUnderstandingProcess</id>
-    <version>21.4.4-preview-test</version>
+    <version>21.4.4-preview</version>
```
Comment (catalin-marin) [inline DocumentUnderstandingProcess/UiPath.Template.DocumentUnderstandingProcess.nuspec:5]: check if this is the correct version ? 
we had a talk about 1.0.0-preview

File: .pipelines/azure-pipelines/azure-pipelines-REFramework.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+variables:
+  - template: .pipelines/variables/common.yml
+  - name: projectPath
+    value: 'REFramework'
+  - name: projectNuspec
+    value: 'UiPath.Template.REFramework.nuspec'
+
+jobs:
+  - template: .pipelines/jobs/ci.yml
```
Comment (catalin-marin) [inline .pipelines/azure-pipelines/azure-pipelines-REFramework.yml]: nok, location is relative to your current location

File: .pipelines/azure-pipelines/azure-pipelines-REFramework.yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+  #   - develop
+  #   - master/REFramework/*
+  # paths:
+  #   include:
+  #   - REFramework/*
+  #   - .pipelines/*
+
+
+variables:
+  - template: .pipelines/variables/common.yml
```
Comment (catalin-marin) [inline .pipelines/azure-pipelines/azure-pipelines-REFramework.yml]: nok, location is relative to your current location
../variables/common.yml

**Review by catalin-marin — CHANGES_REQUESTED** [review]

File: .pipelines/azure-pipelines/azure-pipelines-REFramework_Testing.yml (hunk 1 above)
Comment (swittenberger) [reply .pipelines/azure-pipelines/azure-pipelines-REFramework_Testing.yml]: fixed

File: .pipelines/azure-pipelines/azure-pipelines-REFramework_Testing.yml (hunk 2 above)
Comment (swittenberger) [reply .pipelines/azure-pipelines/azure-pipelines-REFramework_Testing.yml]: fixed

File: DocumentUnderstandingProcess/UiPath.Template.DocumentUnderstandingProcess.nuspec (hunk 3 above)
Comment (swittenberger) [reply DocumentUnderstandingProcess/UiPath.Template.DocumentUnderstandingProcess.nuspec:5]: After some discussion in-house, we agreed to move with versioning identical to the UiPath Studio

File: .pipelines/azure-pipelines/azure-pipelines-REFramework.yml (hunk 5 above)
Comment (swittenberger) [reply .pipelines/azure-pipelines/azure-pipelines-REFramework.yml]: fixed

File: .pipelines/azure-pipelines/azure-pipelines-REFramework.yml (hunk 4 above)
Comment (swittenberger) [reply .pipelines/azure-pipelines/azure-pipelines-REFramework.yml]: fixed

**Review by catalin-marin — APPROVED** [review]
LG

**Pull request merged**